gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ocr2/offchainConfig/write.ts

```diff
@@ -77,10 +77,10 @@ export default class WriteOffchainConfig extends SolanaCommand {
       peerIds: operatorsPeerIds,
       reportingPluginConfig: {
         alphaReportInfinite: config.reportingPluginConfig.alphaReportInfinite,
-        alphaReportPpb: config.reportingPluginConfig.alphaReportPpb,
+        alphaReportPpb: Number(config.reportingPluginConfig.alphaReportPpb),
```

Comment:
Note that `Number()` produces an object, not a literal. I don't know where this is used, but it might break `===` statements. Also, `Number()` can return NaN. Is it worth testing for that in the code?

Reply:
There's a difference between `Number(x)`, that produces a `number` and `new Number(x)`, that gives and instance of `Number`
`NaN` is fine, there's a validation later